packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -406,7 +407,29 @@ module.exports = {
       fileName: 'asset-manifest.json',
       publicPath: publicPath,
     }),
-  ],
+    // TypeScript type checking
+    fs.existsSync(paths.appTSConfig) &&
```

Comment:
nit: can we rename this variable to `appTsConfig` to match the ForkTsChecker variable?

Reply:
Changing ForkTSChecker makes more sense, we use "TS" for TypeScript not "Ts"

Reply:
`appTsConfig` is easier on the eyes :p